src/animalKingdom/MoverInt.java

```diff
@@ -0,0 +1,6 @@
+package animalKingdom;
+
+public interface MoverInt
+{
+	String move();
+}
```

Comment:
I love how you are trying other things like how you made "mover" interface! Way cool!